modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectView/FilterScreen/FilterScreen.tsx

```diff
@@ -16,11 +16,11 @@ import {useModal} from '@clayui/modal';
 import React, {useContext, useState} from 'react';
 
 import {BuilderScreen} from '../BuilderScreen/BuilderScreen';
-import {ModalAddDefaultFilterColumn} from '../ModalAddDefaultFilterColumn/ModalAddDefaultFilterColumn';
+import {ModalAddFilterColumn} from '../ModalAddFilterColumn/ModalAddFilterColumn';
 import ViewContext from '../context';
 
-export function DefaultFilterScreen() {
-	const [{objectView}] = useContext(ViewContext);
+export function FilterScreen() {
+	const [{objectFields, objectView}] = useContext(ViewContext);
```

Comment:
@danvb112 Sorry, I staged some changes for the same file in different commits, but failed in keeping it cohesive... It will be used in a later commit :)